src/views/Predictions/components/Label.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Box, Card, Text } from '@pancakeswap-libs/uikit'
+import { useBnbUsdtTicker } from 'hooks/ticker'
```

Comment:
Just thinking about the error handling in this `useBnbUsdTicket` component being imported. In local testing when this was hit, it crashes the app. If we accept that it will occasionally throw errors, Is there a way to handle this more gracefully for our users... may we need to force a page reload? 

https://github.com/pancakeswap/pancake-frontend/blob/21419163a30e9aa5d98267a609cb3c02d02271da/src/hooks/ticker.ts#L82

```js
      ws.onerror = (error) => {
        console.error('Ticker:', error)
        throw new Error('Ticker: An error occurred')
      }
```

<img width="1103" alt="Screenshot 2021-04-26 at 12 44 30" src="https://user-images.githubusercontent.com/79279477/116082150-fde4d300-a692-11eb-9f53-3d92e04c4417.png">